packages/extension-api/src/extension-api.d.ts

```diff
@@ -2101,6 +2101,51 @@ declare module '@podman-desktop/api' {
     diskUsed?: number;
   }
 
+  export interface VolumeInfo {
+    engineId: string;
+    engineName: string;
+    CreatedAt: string;
+    containersUsage: { id: string; names: string[] }[];
+    Name: string;
+    Driver: string;
+    Mountpoint: string;
+    Status?: { [key: string]: string };
+    Labels: { [key: string]: string };
+    Scope: 'local' | 'global';
+    Options?: { [key: string]: string } | null;
```

Comment:
The usage of null is not supposed to be prohibited ? Is this definition from the kube api itself ?

Reply:
definition comes from 3rd party libs (where null is not prohibited)

Reply:
so if we return an object not matching the definition, there is a conflict